CONTRIBUTING.md

```diff
@@ -116,23 +116,35 @@ and the final version are only comitted and tagged in this branch. The master br
 development versions, allowing us to keep the branches completely separate without merging releases
 back into master.
 
-The exact steps for an exemplary `1.2.3` release might look like this:
- 1. Initially, the version on the latest master is `1.2.3-dev`
- 2. A new `v1.2.3` branch is created for the release
- 3. On master, the version is bumped to `1.2.4-dev`
- 4. In the branch, the version is bumped to `1.2.3-rc1`
- 5. The new commit in the branch is tagged as `v1.2.3-rc1`
- 6. A GitHub release is created for the `v1.2.3-rc1` tag
- 7. The changelog since the last release (stable or RC)
-        is added to the GitHub release description
- 8. Bug fixes are cherry-picked from master into the branch and steps 4-7
-        are repeated until no major issues are found in the release candidates
- 9. In the branch, the version is bumped to `1.2.3`
- 10. The new commit in the branch is tagged as `v1.2.3`
- 11. A GitHub release is created for the `v1.2.3` tag
- 12. The changelog since the last stable release (**not** RC)
-        is added to the GitHub release description
- 13. The `-dev` is stripped from previous changelog entries
+The exact steps for an exemplary `0.2.0` release might look like this:
+  1. Initially, the version on the latest master is `0.2.0-dev`
+  2. A new `v0.2.0` branch is created for the release
+  3. On master, the version is bumped to `0.3.0-dev`
+  4. In the branch, the version is bumped to `0.2.0-rc1`
+  5. The new commit in the branch is tagged as `v0.2.0-rc1`
+  6. A GitHub release is created for the `v0.2.0-rc1` tag
+  7. The changelog since the last release (stable or RC) is added to the GitHub release description
+  8. Bug fixes are cherry-picked from master into the branch and steps 4-7 are repeated until no
+     major issues are found in the release candidates
+  9. In the branch, the version is bumped to `0.2.0`
+ 10. The new commit in the branch is tagged as `v0.2.0`
+ 11. A GitHub release is created for the `v0.2.0` tag
+ 12. The changelog since the last stable release (**not** RC) is added to the GitHub release
+     description
+ 13. The `-dev` is stripped from the `0.2.0-dev` changelog entries on master
+
+On the master and with new planned releases, the patch version is never bumped, but only the minor
+is increased. This allows for creating releases to fix bugs which build on previous minor releases
+and just increment the patch version, without having to decrement or advance the version based on
+these unplanned releases.
```

Comment:
```suggestion
On master and with new planned releases, the patch version is never bumped; only the minor version
is increased. This allows for creating releases to fix bugs which build on top of previous minor releases
and just increment the patch version.
```

I think the last part I removed is unnecessary. The preceding portion of the sentence already gives enough context on why this should be done this way, there's no reason to bring in decrementing versions, etc., IMHO.

(Sorry, you'll have to reflow yourself if you accept this...)

Reply:
The last part is not irrelevant though, because you could build on top of previous minor releases and just increment the patch version even if you bumped the patch on master too.

The difference is just that you do not have to deal with incrementing or decrementing the master, which is why I explicitly pointed this out.